Models/api.model.js

```diff
@@ -77,3 +77,29 @@ exports.postNewComment = ({ username, body }, article_id) => {
       return result.rows[0];
     });
 };
+
+exports.votePatchAdd = (article_id, inc_votes) => {
+  return db.query(`
+    UPDATE articles
+    SET votes = votes + $1
```

Comment:
If inc_votes was a negative then it would reduce the votes by the correct amount even if adding. no need for positive and negatives to be handled separately